packages/migrate/src/interfaces/fix-transform.ts

```diff
@@ -1,17 +1,27 @@
 import ts from 'typescript';
+import RehearsalService from '../rehearsal-service';
 
 export interface FixedFile {
   fileName: string;
   text: string;
 }
+
 export default class FixTransform {
   /** Engineer friendly message describes steps needs to be done to fix the related issue */
   hint?: string;
 
   /** Function to fix the diagnostic issue */
-  fix?: (diagnostic: ts.DiagnosticWithLocation, service: ts.LanguageService) => FixedFile[];
+  fix?: (
+    diagnostic: ts.DiagnosticWithLocation,
+    bathPath: string,
+    service: RehearsalService
+  ) => FixedFile[];
 
-  run(diagnostic: ts.DiagnosticWithLocation, service: ts.LanguageService): FixedFile[] {
-    return this.fix ? this.fix(diagnostic, service) : [];
+  run(
+    diagnostic: ts.DiagnosticWithLocation,
+    basePath: string,
```

Comment:
I feel the `service` is a good place to hold `basePath`... can event be implemented as a getter to grab the "calculated" base path from compilerOptions.

Reply:
Removed basePath.